extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthConfig.java

```diff
@@ -42,6 +42,12 @@
     @ConfigItem(defaultValue = "well")
     String wellnessPath;
 
+    /**
+     * Whether the context should be propagated to each health check invocation.
+     */
+    @ConfigItem(defaultValue = "false")
```

Comment:
I removed the enforced name here as it's the default that will be generated so no need to define it.

Reply:
ah, ok, didn't know, thanks.